IDEAS/Templates/BaseCircuits/PumpSupply_dp.mo

```diff
@@ -20,7 +20,6 @@ model PumpSupply_dp
       riseTime=riseTime,
       init=init,
       massDynamics=Modelica.Fluid.Types.Dynamics.SteadyState,
-      motorCooledByFluid=false,
```

Comment:
@GlennReynders is it on purpose that you want to remove the `motorCooledByFluid`?